v5/pivotal/projects.go

```diff
@@ -116,3 +116,39 @@ func (service *ProjectService) Get(projectId int) (*Project, *http.Response, err
 
 	return &project, resp, err
 }
+
+func (service *ProjectService) AddStory(
```

Comment:
Hey, sorry for silence.  Been pretty busy lately and just wanted to let you know it might take a couple more weeks before I get time for this again. Will try to fix up based on your feedback at that point though.